Resita/index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="../Resita/css/maincss.css">
+    <link href="https://fonts.googleapis.com/css?family=Dancing+Script|Indie+Flower" rel="stylesheet">
+    <link rel="icon" href="../Resita/favicon-16x16.png" type="image/x-icon" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+        integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <title>Resita</title>
+</head>
+
+<body>
+    <div id="top">
+        <ul>
+            <li id="emblem">
+                        <img src="../Resita/images/ROU_CS_Resita_CoA.png" alt="Resita Coat of Arms">
+            </li>
+            <li>
+                <a href="#history">
+                    <i class="fas fa-monument"></i> <span> History</span></a>
+                </li>
+            <li>
+                <a href="#location">
+                    <i class="fas fa-map-marker-alt"></i><span> Location</span></a>
+                </li>
+            <li>
+                <a href="#images">
+                    <i class="fas fa-image"></i><span> Images</span></a>
+                </li>
+            <li>
+                <a href="#sights">
+                    <i class="fas fa-camera"></i><span> Sights</span></a>
+                </li>
+            
+        </ul>
+    </div>
+    <div class="box1">
+        <img src="../Resita/images/resita2.jpg" alt="Resita at Night">
+        <h1>RESITA</h1>
+    </div>
+    <div class="h2">
```

Comment:
Avoid using classnames that might refer to other HTML elements. Instead of h2 that refers to heading styling you can use title for your h2 and 'content' for box2
So you will have a title div and a content div inside your section named row or something